packages/console/src/components/PlanUsage/PlanUsageCard/index.tsx

```diff
@@ -1,42 +1,62 @@
 import { type AdminConsoleKey } from '@logto/phrases';
 import { conditional, type Nullable } from '@silverhand/essentials';
 import classNames from 'classnames';
+import { useContext } from 'react';
 import { Trans, useTranslation } from 'react-i18next';
 
 import Tip from '@/assets/icons/tip.svg?react';
 import { addOnPricingExplanationLink } from '@/consts/external-links';
+import { tokenCountUnit } from '@/consts/subscriptions';
+import { SubscriptionDataContext } from '@/contexts/SubscriptionDataProvider';
 import DynamicT from '@/ds-components/DynamicT';
 import IconButton from '@/ds-components/IconButton';
 import Tag from '@/ds-components/Tag';
 import TextLink from '@/ds-components/TextLink';
 import { ToggleTip } from '@/ds-components/Tip';
+import { isPaidPlan } from '@/utils/subscription';
 
 import { formatNumber } from '../utils';
 
 import styles from './index.module.scss';
 
+const formatQuotaNumber = (rawNumber: number): string => {
+  const number = rawNumber >= 0.1 * tokenCountUnit ? rawNumber / tokenCountUnit : rawNumber;
+
+  if (Number.isInteger(number)) {
+    return number.toString();
+  }
+  return number.toFixed(2);
+};
```

Comment:
this looks ambiguous. if we want to show a better format, we need to also handle thousand numbers like "1K" as well?

Reply:
Can do, the initial purpose of wrapping this function was to simply demonstrate the token usage limit, which is a number on the order of millions. These numbers can also be rewritten using kilo as the unit.